doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -2,111 +2,75 @@
+This means you already have a repository of ingested DECam data and have set up the LSST Science Pipelines stack.
 
-Your directory structure should look something like
+Your repository should have the following collections, which can be checked using ``butler query-collections <repo>``:
 
-.. code-block:: none
+- **calib/DECam**: biases, flats, defects, camera specs, etc.
+- **raw/DECam**: images to be processesd
+- **refcats**: reference catalogs for calibration
+- **skymaps**: index for the templates
+- **templates/deep**: difference imaging templates
 
-   repo/   <-- where you will be running ap_pipe
-      registry.sqlite3   <--- created during image ingestion
-      image1.fits   <--- linked here by default during image ingestion
-      image2.fits
-      ...
-      ref_cats/   <--- copied or linked here manually by you
-         gaia/
-            shard1
-            shard2
-            ...
-         pan-starrs/
-            shard1
-            shard2
-            ...
-      calibs/
-         calibRegistry.sqlite3   <--- created/updated during calib/defect ingestion
-         cpBIAS/
-            bias1.fits   <--- linked here by default during calib ingestion
-            bias2.fits
-            ...
-         cpFLAT/
-            flat1.fits   <--- linked here by default during calib ingestion
-            flat2.fits
-            ...
-         defects/   <--- put here manually by you prior to defect ingestion
-            defect1.fits
-            defect2.fits
-            ...
-      templates/   <--- copied or linked here manually by you
-         repositoryCfg.yaml
-         deepCoadd/
-            g/
-               0/
-                  psfMatched-0,0.fits
-                  psfMatched-0,1.fits
-                  ...
 
 .. _section-ap-pipe-command-line:
 
 AP pipeline on the command line
 ===============================
 
-The executable to run for the AP Pipeline (`ApPipeTask`) is in ``ap_pipe/bin/ap_pipe.py``.
+Like most Vera Rubin Observatory pipelines, the AP Pipeline is run with an external runner called ``pipetask``.
+This can be found in the ``ctrl_mpexec`` package, which is included as part of ``lsst_distrib``.
+
+The pipeline itself is configured in ``ap_pipe/pipelines/ApPipe.yaml``.
 
 To process your ingested data, run
 
 .. prompt:: bash
 
    mkdir apdb/
    make_apdb.py -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED -c diaPipe.apdb.db_url="sqlite:///apdb/association.db"
-   ap_pipe.py repo --calib repo/calibs --rerun processed -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED -c diaPipe.apdb.db_url="sqlite:///apdb/association.db" --id visit=123456 ccdnum=42 filter=g --template templates
+   pipetask run -p ${AP_PIPE_DIR}/pipelines/ApPipe.yaml --instrument lsst.obs.decam.DarkEnergyCamera --register-dataset-types -c diaPipe:apdb.isolation_level=READ_UNCOMMITTED -c diaPipe:apdb.db_url="sqlite:///apdb.db" -b repo/ -i "templates/deep,skymaps,raw/DECam,calib/DECam,refcats" -o processed -d "visit=123456 and detector=42"
```

Comment:
Please update so it does two visits, 123456 and 123457, because otherwise the APDB will have no DIA-anything

Reply:
Hmm... does it matter if the two visits overlap? I'm guessing consecutive ones don't.

Reply:
They do need to overlap spatially, yes. More than once I've had a random user ask me questions about why their APDB is empty when they ran ap_pipe on a single dataId, so my thinking is that including 2 visits in the example at least theoretically shows the need to run it on 2+ things.

Reply:
Right, these are fake visit IDs anyway. 🤦